storage_s3/indico_storage_s3/storage.py

```diff
@@ -108,8 +109,18 @@ def _save(self, bucket, name, content_type, fileobj):
         checksum = get_file_checksum(fileobj)
         fileobj.seek(0)
         content_md5 = checksum.decode('hex').encode('base64').strip()
-        self.client.put_object(Body=fileobj, Bucket=bucket, Key=name,
-                               ContentType=content_type, ContentMD5=content_md5)
```

Comment:
ContentMD5 is actually added automatically by botocore, so there's no need to manually pass it (and it wouldn't work for chunked uploads anyway since the md5 here is not the md5 on the individual chunks)